### ISSUE TITLE
Test failure: MvcTemplate_IndividualAuth_Works_NetFramework

### ISSUE THREAD

**Issue by aspnet-hello** [aspnet/Templating#706]
This test [fails](http://aspnetci/viewLog.html?buildId=537992&buildTypeId=Lite_UniverseTest) occasionally with the following error:
```
System.Exception : Process exited with code 1
StdErr: 
StdOut: Build failed.
   at Templates.Test.Helpers.ProcessEx.WaitForExit(Boolean assertSuccess) in /_/test/Templates.Test/Helpers/ProcessEx.cs:line 150
   at Templates.Test.TemplateTestBase.RunDotNetEfCreateMigration(String migrationName) in /_/test/Templates.Test/Helpers/TemplateTestBase.cs:line 137
   at Templates.Test.MvcTemplateTest.MvcTemplate_IndividualAuthImpl(String targetFrameworkOverride, Boolean useLocalDB, Boolean noHttps) in /_/test/Templates.Test/MvcTemplateTest.cs:line 123
------- Stdout: -------
==> C:\b\w\33bdfc1cae7b2a38\.dotnet\x64\dotnet.exe new mvc --debug:custom-hive "C:\b\w\33bdfc1cae7b2a38\modules\Templating\test\Templates.Test\bin\Release\netcoreapp3.0\.templateengine" --target-framework-override net461 --auth Individual [C:\b\w\33bdfc1cae7b2a38\modules\Templating\test\Templates.Test\bin\Release\netcoreapp3.0\TestTemplates\AspNet.Template.7e9fb79e946949f78af5ac2e3bc335cf]
The template "ASP.NET Core Web App (Model-View-Controller)" was created successfully.
This template contains technologies from parties other than Microsoft, see https://aka.ms/aspnetcore-template-3pn-210 for details.
Processing post-creation actions...
Running 'dotnet restore' on C:\b\w\33bdfc1cae7b2a38\modules\Templating\test\Templates.Test\bin\Release\netcoreapp3.0\TestTemplates\AspNet.Template.7e9fb79e946949f78af5ac2e3bc335cf\AspNet.Template.7e9fb79e946949f78af5ac2e3bc335cf.csproj...
  Restoring packages for C:\b\w\33bdfc1cae7b2a38\modules\Templating\test\Templates.Test\bin\Release\netcoreapp3.0\TestTemplates\AspNet.Template.7e9fb79e946949f78af5ac2e3bc335cf\AspNet.Template.7e9fb79e946949f78af5ac2e3bc335cf.csproj...
  Generating MSBuild file C:\b\w\33bdfc1cae7b2a38\modules\Templating\test\Templates.Test\bin\Release\netcoreapp3.0\TestTemplates\AspNet.Template.7e9fb79e946949f78af5ac2e3bc335cf\obj\AspNet.Template.7e9fb79e946949f78af5ac2e3bc335cf.csproj.nuget.g.props.
  Generating MSBuild file C:\b\w\33bdfc1cae7b2a38\modules\Templating\test\Templates.Test\bin\Release\netcoreapp3.0\TestTemplates\AspNet.Template.7e9fb79e946949f78af5ac2e3bc335cf\obj\AspNet.Template.7e9fb79e946949f78af5ac2e3bc335cf.csproj.nuget.g.targets.
  Restore completed in 842.47 ms for C:\b\w\33bdfc1cae7b2a38\modules\Templating\test\Templates.Test\bin\Release\netcoreapp3.0\TestTemplates\AspNet.Template.7e9fb79e946949f78af5ac2e3bc335cf\AspNet.Template.7e9fb79e946949f78af5ac2e3bc335cf.csproj.
Restore succeeded.
==> C:\b\w\33bdfc1cae7b2a38\.dotnet\x64\dotnet.exe "C:\b\w\33bdfc1cae7b2a38\packages\dotnet-ef/3.0.0-alpha1-10346/tools/netcoreapp3.0/any/dotnet-ef.dll" migrations add mvc [C:\b\w\33bdfc1cae7b2a38\modules\Templating\test\Templates.Test\bin\Release\netcoreapp3.0\TestTemplates\AspNet.Template.7e9fb79e946949f78af5ac2e3bc335cf]
Build failed.
```
Other tests within that build may have failed with a similar message, but they are not listed here. Check the link above for more info.

This test failed on master.

CC @mkArtakMSFT

This issue was made automatically. If there is a problem contact ryanbrandenburg.

**Comment by aspnet-hello** [comment]
Please use this workflow to address this flaky test issue, including checking applicable checkboxes and filling in the applicable "TODO" entries:

* Is this actually a flaky test?
  * No, this is a regular test failure, fix the test/product (TODO: Link to commit/PR)
  * Yes, proceed below...

* Is this test failure caused by product code flakiness? (Either this product, or another product this test depends on.)
  * [ ] File a bug against the product (TODO: Link to other bug)
  * Is it possible to change the test to avoid the flakiness?
    * Yes? Go to the "Change the test!" section.
    * No?
      * [ ] Disable the test (TODO: Link to PR/commit)
      * [ ] Wait for other bug to be resolved
      * [ ] Wait for us to get build that has the fix
      * [ ] Re-enable our test (TODO: Link to PR/commit)
      * [ ] Close this bug

* Is it that the test itself is flaky? This includes external transient problems (e.g. remote server problems, file system race condition, etc.)
  * Is there is a way to change our test to avoid this flakiness?
    * Yes? Change the test!
      * [ ] Change the test to avoid the flakiness, for example by using a different test strategy, or by adding retries w/ timeouts (TODO: Link to PR/commit)
      * [ ] Run the test 100 times locally as a sanity check.
      * [ ] Close this bug
    * No?
      * Is there any logging or extra information that we could add to make this more diagnosable when it happens again?
        * Yes?
            * [ ] Add the logging (TODO: Link to PR/commit)
        * No?
            * [ ] Delete the test because flaky tests are not useful (TODO: Link to PR/commit)

This comment was made automatically. If there is a problem contact ryanbrandenburg.

**Comment by aspnet-hello** [comment]
RazorPagesTemplate_WithIndividualAuth_NoHttpsSetToTrue_UsesHttps_NetFramework [failed with about the same error](http://aspnetci/viewLog.html?buildId=537992&buildTypeId=Lite_UniverseTest) on master.

This comment was made automatically. If there is a problem contact ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
The problem here is that our template fails to compile because we consumed a new Enum from Identity in 3.0 before it was merged forward in the Identity repo. This will be fixed when Identity merges aspnet/Identity#1933. I would do it myself but there are currently test failures and I don't know if it's in a good state.

Is problem only affects 3.0, 2.2 has already been merged.

**Comment by mkArtakMSFT** [comment]
Done per @ryanbrandenburg.
